ex02.ex

```diff
@@ -41,7 +41,7 @@ defmodule Ex02 do
   # Do the same using the & syntax #
   ##################################
 
-  list2b = your_anonymous_function(1, 2)
+  list2b = &([&1 + &2, &1 - &2])
```

Comment:
No deduction, but: the unneeded parens make the expression harder to read. You could write:

~~~ elixir
list2b = &[ &1 + &2, &1 - &2 ]
~~~

Notice I also added a space inside the []
s